src/Apps/Artwork/Components/ArtworkDetails/__tests__/ArtworkDetails.test.tsx

```diff
@@ -50,6 +50,41 @@ describe("ArtworkDetails", () => {
   }
   let wrapper
 
+  describe("ArtworkDetailsAdditionalInfo for a live sale artwork", () => {
+    it("displays a request lot condition report button when canRequestLotConditionsReport is true", async () => {
+      wrapper = await getWrapper({
+        ...ArtworkDetailsFixture,
+        canRequestLotConditionsReport: true,
+      })
+
+      expect(wrapper.html()).toContain("Condition")
```

Comment:
Is there some text that we can assert upon that's a bit closer to the button. Perhaps "Request condition report"?